packages/core/src/components/Dialog/Title/Title.styles.tsx

```diff
@@ -1,31 +1,14 @@
-import styled from "@emotion/styled";
 import { theme } from "@hitachivantara/uikit-styles";
-import DialogTitle from "@mui/material/DialogTitle";
-import { transientOptions } from "@core/utils/transientOptions";
 
-export const StyledTitle = styled(
-  DialogTitle,
-  transientOptions
-)(({ $fullscreen }: { $fullscreen: boolean }) => ({
-  padding: theme.space.sm,
-  margin: 0,
-  ...(!$fullscreen && {
-    flex: 1,
-  }),
-}));
+import { createClasses } from "@core/utils";
 
-export const StyledMessageContainer = styled("div")({
-  display: "flex",
-  alignItems: "center",
-});
-
-export const StyledTextWithIcon = styled(
-  "div",
-  transientOptions
-)(({ $hasIcon }: { $hasIcon: boolean }) => ({
-  ...($hasIcon && {
+export const { staticClasses, useClasses } = createClasses("HvDialog-Title", {
+  root: { padding: theme.space.sm, margin: 0 },
+  fullscreen: {},
+  messageContainer: { display: "flex", alignItems: "center" },
```

Comment:
I like condensed objects - fewer lines of code! But on the other side, once we add/delete/modify a single style, we'll have to read/git diff the whole class - and CSS is scary 😝